packages/astro/test/i18n-routing.test.js

```diff
@@ -661,7 +663,9 @@ describe('[SSG] i18n routing', () => {
 						fallback: {
 							it: 'en',
 						},
-						routingStrategy: 'prefix-always',
+						routing: {
+							prefixDefaultLocale: false,
```

Comment:
If this should mirror the previous setting, I guess it should be `true`?
```suggestion
							prefixDefaultLocale: true,
```